azure-pipelines.yml

```diff
@@ -70,7 +70,13 @@ stages:
       - task: NuGetToolInstaller@1
         displayName: "Install NuGet"
 
+      - script: 'dotnet nuget locals all --clear'
+        displayName: "NuGet Cache Clear"
```

Comment:
This is to ensure there is no NuGet cache.
I will remove this change before merging.